frontend/src/components/movie-card-content/MovieCardContent.tsx

```diff
@@ -63,11 +73,13 @@ function MovieCardContent({
         <MovieDetailsRight>
           <Rating>
             <StarIcon sx={{ color: FRENCH_MAUVE }} />
-            <Score>{calculateAverageRating(getReviews(reviews))} / 5</Score>
+            <Score>{calculateAverageRating(getReviews(reviews))}</Score>
           </Rating>
-          <DeleteMovieButton onClick={handleButton}>
```

Comment:
You can add an empty line between children in MovieDetailsRight for better visibility.